test/functional/test_runner.py

```diff
@@ -422,17 +436,20 @@ def __init__(self, name, status, time):
         self.padding = 0
 
     def __repr__(self):
-        if self.status == "Passed":
+        if self.status == STATUS_PASSED:
             color = BLUE
             glyph = TICK
-        elif self.status == "Failed":
-            color = RED
-            glyph = CROSS
-        elif self.status == "Skipped":
+        if self.status == STATUS_PASSED_WITH_WARNINGS:
+            color = MAGENTA
+            glyph = TICK
```

Comment:
Is TICK the best glyph to use for passed with warnings? Or something more neutral?